client/app/telecom/pack/move/eligibility/address/pack-move-eligibility-address.component.js

```diff
@@ -36,7 +36,7 @@ angular.module("managerApp").component("packMoveEligibilityAddress", {
                         self.address.city = self.cities[0];
                     }
                 }, function () {
-                    return new ToastError($translate.instant("pack_move_eligibility_zipcode_error", { zipcode: zipcode }));
+                    return Toast.error($translate.instant("pack_move_eligibility_zipcode_error", { zipcode: zipcode }));
```

Comment:
It's preferable to reject the promises.

1. first display the toas error ;
2. then `return $q.reject(error);`